README.md

```diff
@@ -26,9 +26,33 @@ Documentation for this package is also available at:
 
 * https://pkg.go.dev/github.com/seccomp/libseccomp-golang
 
+## Verifying Releases
+
+Starting with libseccomp-golang v0.9.2, the git tag corresponding to each
```

Comment:
nit: perhaps we should rename 0.9.2 to 0.10.0, since there are quite a few changes in there already (seccomp-notify support alone is quite enough to increment a minor version).

Reply:
That's a pretty reasonable point, I'd go along with it.  What do you think @drakenclimber, skip v0.9.2 in favor of 0.10.0?

Reply:
That's a compelling argument to me.  `0.10.0` it is!

Reply:
Wow, you guys are quick, no sooner did I just create an issue to track this and now I see it's been decided :)

I'll make those changes now - thanks guys!